map/src/context/RoutingManager.js

```diff
@@ -102,60 +103,439 @@ function filterMode(data) {
     return Object.fromEntries(Object.entries(data).filter(([key]) => !key.includes('rescuetrack')));
 }
 
+function osrmToFeaturesCollection(osrm) {
```

Comment:
please separate providers to new manager file

Reply:
> please separate providers to new manager file

routeProviders will be separated later, in the next routerClass release